### DISCUSSION TITLE
frugen: command not found

### DISCUSSION THREAD

**Discussion by yohey55** [ipmitool/frugen#20] [Q&A]
Hi everyone

I use Raspberry Pi 4B and Raspbian as OS.
The following is my process of installing frugen library:

git clone https://github.com/ipmitool/frugen
cmake .
make

and then it looks cmake and make ok:
[ 25%] Built target fru-shared
[ 50%] Built target fru-static
[ 75%] Built target frugen
[100%] Built target frugen-static

But when I try: frugen -h or any "frugen" command
it shows: -bash: frugen: command not found

What's wrong with it?

**Comment by AlexanderAmelkin** [comment]
Have you checked this?
https://askubuntu.com/questions/1004373/running-a-program-from-terminal-command-not-found#1004377
